Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
 }
 
-// MARK: - GitHubCommit
+extension GitHub {
+    public struct Repo: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case id
+            case name
+            case owner
+            case description
+            case fullName = "full_name"
+            case isPrivate = "private"
+            case isFork = "fork"
+            case htmlURL = "html_url"
+        }
 
-/// A GitHub specific implementation of a git commit.
-public struct GitHubCommit: Decodable, Equatable {
-    // MARK: - Properties
-
-    enum CodingKeys: String, CodingKey {
-        case sha
-        case commit
-        case url
-        case author
-        case committer
-    }
+        /// Generic UUID.
+        public let id: Int
 
-    /// The SHA for the commit.
-    public let sha: String
+        /// The name of the repo, e.g. "danger-swift".
+        public let name: String
 
-    /// The raw commit metadata.
-    public let commit: GitCommit
+        /// The full name of the owner + repo, e.g. "Danger/danger-swift"
+        public let fullName: String
 
-    /// The URL for the commit on GitHub.
-    public let url: String
+        /// The owner of the repo.
+        public let owner: User
 
-    /// The GitHub user who wrote the code.
-    public let author: GitHubUser?
+        /// A boolean stating whether the repo is publicly accessible.
+        public let isPrivate: Bool
 
-    /// The GitHub user who shipped the code.
-    public let committer: GitHubUser?
+        /// A textual description of the repo.
+        public let description: String?
 
-    public init(from decoder: Decoder) throws {
-        let container = try decoder.container(keyedBy: CodingKeys.self)
+        /// A boolean stating whether the repo is a fork.
+        public let isFork: Bool
 
-        sha = try container.decode(String.self, forKey: .sha)
-        commit = try container.decode(GitCommit.self, forKey: .commit)
-        url = try container.decode(String.self, forKey: .url)
-        author = (try? container.decodeIfPresent(GitHubUser.self, forKey: .author)) ?? nil
-        committer = (try? container.decodeIfPresent(GitHubUser.self, forKey: .committer)) ?? nil
+        /// The root web URL for the repo, e.g. https://github.com/artsy/emission
+        public let htmlURL: String
     }
 }
 
-// MARK: - GitHubIssue
-
-public struct GitHubIssue: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case number
-        case title
-        case user
-        case state
-        case assignee
-        case assignees
-        case milestone
-        case body
-        case labels
-        case commentCount = "comments"
-        case isLocked = "locked"
-        case createdAt = "created_at"
-        case updatedAt = "updated_at"
-        case closedAt = "closed_at"
+extension GitHub {
+    public struct Review: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
```

Comment:
<!--
  0 failure: 
  1 warning:  Types should be n...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 244;
-->

- :warning: Types should be nested at most 1 level deep (`nesting`)